src/content/docs/journey-app-slow/root-causes.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Prepare to triage your application
+metaDescription: 
+---
+
+Your app is slow. Now what? How do you track down the root cause of your app's slowness?
+
+This tutorial walks you through how to use New Relic to triage your application and identify the root cause of your app's degraded performance. You'll start by "instrumenting" your app with an agent, which means installing a piece of code that reports data about your app to New Relic. Using this data in New Relic, you'll dig into the performance of your application and identify slow transactions, slow database queries, or slow external services as your issue - or possibly all three!
```

Comment:
I almost want to put 'This tutorial' sentence at the end of the rhetorical line above, then have this paragraph break start with 'You'll start by instrumenting' blahblahblah